plugins/UsersManager/templates/userSettings.twig

```diff
@@ -107,6 +107,15 @@
     </form>
 </div>
 
+<div piwik-content-block
+     content-title="{{ 'UsersManager_TokenAuth'|translate|e('html_attr') }}">
+    <p id="token_auth_user" data-token="{{ userTokenAuth }}">{{ userTokenAuth }}</p>
+
+    <button class="btn btn-link"
+            ng-controller="PersonalSettingsController as personalSettings"
+            ng-click="personalSettings.regenerateTokenAuth()">{{ 'UsersManager_RegenerateTitle'|translate }}</button>
```

Comment:
I have no clue honestly, I don't mind people being logged out we would only need to mention it